docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,306 @@
+| `trigger_1_confidence` | float | Optional | The detection confidence needed in order to move into the `TRIGGER_2` state. <br> Default: `0.2` |
+| `trigger_2_detector` | string | **Required** | The name of the vision service detector that will detect the thing that needs to be verified. |
+| `trigger_2_labels` | array | **Required** | The class names from `trigger_2_detector` that count as valid. |
+| `trigger_2_confidence` | float | Optional | The detection confidence needed in order to move into the `COUNTDOWN` state. <br> Default: `0.5` |
+| `verification_detector` | string | **Required** | The name of the vision service detector that you want to use to verify the object. |
+| `verification_labels` | array | **Required** | The class names from `verification_detector` that count as valid. |
+| `verification_confidence` | float | Optional | The detection confidence needed in order to move into the `DISARMED` state. <br> Default: `0.8` |
+| `countdown_time_s` | int | Optional | The time in seconds the system will remain in state `COUNTDOWN` before transitioning to state `ALARM`. <br> Default: `20` |
+| `alarm_time_s` | int | Optional | The time in seconds the system will remain in  state `ALARM` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disarmed_time_s` | int | Optional | The time in seconds the system will remain in  state `DISARMED` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disable_alarm` | bool | Optional | Disables the `COUNTDOWN` and `ALARM` states. The system will always remain in the `TRIGGER_1` and `TRIGGER_2` states. <br> Default: `false` |
+
+Note that `ALARM` is by default silent and only a label overlay, but you can [poll the classifications](/ml/vision/#getclassificationsfromcamera) to set up a physical alarm trigger.
+
+## Configure a transform camera
+
+To view the classifications that your verification system makes, configure a transform camera.
+
+Navigate to the **Config** tab of your machine’s page in [the Viam app](https://app.viam.com/).
+Click on the **Components** subtab and click **Create component**. Select the `camera` type, then select the `transform` model. Enter a name for your camera and click **Create**.
+
+With **Builder** mode selected, copy and paste the following attributes JSON into your camera's **Attributes** box:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "pipeline": [
+    {
+      "type": "classifications",
+      "attributes": {
+        "classifier_name": "security",
+        "confidence_threshold": 0.5
+      }
+    }
+  ],
+  "source": "my-webcam"
+}
+```
+
+Note that the camera name in `"source"` and the `"classifier_name"` match the names that we configured for our source camera and verification system service, respectively.
+If you used different names, edit as applicable.
+View these [instructions to configure a transform camera](/components/camera/transform/) for reference.
+Save your config.
```

Comment:
Nit: let's convert this to steps as well:

````suggestion
To add a transform camera to your machine:

1. On your machine's **Config** page in the [Viam app](https://app.viam.com), navigate to the **Components** tab.
1. Click the **Create component** button at the bottom of the page, select `camera`, then select the built-in `transform` model.
1. Give the transform camera a name, like `my-transform-camera`, then click **Create**.
1. On the panel that appears, enter the following configuration into the **Attributes** field:

   ```json {class="line-numbers linkable-line-numbers"}
   {
     "pipeline": [
       {
         "attributes": {
           "classifier_name": "security",
           "confidence_threshold": 0.5
         },
         "type": "classification"
       }
     ],
     "source": "my-webcam"
   }
   ```

   If you used different names for the vision service or the camera component, update this configuration with those names.
   You can adjust the `confidence_threshold` to suit your needs.
   A value of `0.5` is a relatively loose match, representing 50% confidence.

1. Click **Save Config** at the bottom of the window to save your changes.
````